src/ParseQuery.js

```diff
@@ -1005,21 +1005,49 @@ class ParseQuery {
   *
   * @param {String} key The key that the string to match is stored in.
   * @param {String} value The string to search
+  * @param {Object} options (Optional) See https://docs.mongodb.com/manual/reference/operator/query/text/#text-query-operator-behavior
+  * @param {String} options.language
+  * @param {String} options.caseSensitive
+  * @param {String} options.diacriticSensitive
```

Comment:
Diacritic and case are Boolean type. What happens if they aren't boolean 🤔?

Instead of the mongo docs, put a summary next to the options for readability

Reply:
Like this?

```
/**
  * Adds a constraint for finding string values that contain a provided
  * string. This may be slow for large datasets. Requires Parse-Server > 2.5.0
  *
  * In order to sort you must use select and ascending ($score is required)
  *  <pre>
  *   query.fullText('field', 'term');
  *   query.ascending('$score');
  *   query.select('$score');
  *  </pre>
  *
  * To retrieve the weight / rank
  *  <pre>
  *   object->get('score');
  *  </pre>
  *
  * You can define optionals by providing an object as a third parameter
  *  <pre>
  *   query.fullText('field', 'term', { language: "spanish", diacriticSensitive: true });
  *  </pre>
  *
  * @param {String} key The key that the string to match is stored in.
  * @param {String} value The string to search
  * @param {Object} options (Optional) See https://docs.mongodb.com/manual/reference/operator/query/text/#text
  * @param {String} options.language The language that determines the list of stop words for the search and the rules for the stemmer and tokenizer.
  * @param {Boolean} options.caseSensitive A boolean flag to enable or disable case sensitive search.
  * @param {Boolean} options.diacriticSensitive A boolean flag to enable or disable diacritic sensitive search.
  * @return {Parse.Query} Returns the query, so you can chain this call.
  */
```